### DISCUSSION TITLE
代码修改，请教

### DISCUSSION THREAD

**Discussion by zxhhtt** [XIU2/CloudflareSpeedTest#432] [提问]
大佬，你好。
我fork了你的项目，做了一些修改。
遇到个问题：我想修改默认的测速地址，以后不用带-url参数，
我改了task里的download，还有main以及go.mod等等，
但是，没实现。希望大佬指点一下

**Comment by XIU2** [comment]
你直接代码里搜索 `https://cf.xiu2.xyz/url` 全部替换为你的不就行了？

**Comment by zxhhtt** [comment]
好了，谢谢大佬，交作业。
![Snipaste_2023-09-11_19-09-51](https://github.com/XIU2/CloudflareSpeedTest/assets/140737998/798cf897-4d5a-4f49-a152-163cbb62c3a7)
![Snipaste_2023-09-11_19-10-25](https://github.com/XIU2/CloudflareSpeedTest/assets/140737998/d2c0fa18-a760-4155-be7b-88891f290334)
